install-daemon.sh

```diff
@@ -22,7 +22,7 @@ fi
 CURLPATH="$(command -v curl)"
 if [ -z "$CURLPATH" ]; then
     echo "* curl is required in order for this script to work."
-    echo "* install using apt on Debian/Ubuntu or yum on CentOS"
+    echo "* install using apt on Debian/Ubuntu/Zorin or yum on CentOS"
```

Comment:
I think people would understand that `apt` is available on Zorin OS as well (not necessary to put here).